src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -162,6 +191,26 @@ public void verifyInstructorEditFormDisabled(int instrNum) {
         assertTrue(isNotEditable);
     }
 
+    /**
+     * Verifies that the fields contain the updated values.
+     * If {@code newIsDisplayedToStudents} is true, the field's value is checked against {@code newDisplayName}.
```

Comment:
`the display name field is checked against {@code newDisplayName}` instead? What `the field` is referring to is ambiguous.